docker-compose.yaml

```diff
@@ -18,11 +18,11 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_DB: data_store_api
 
-  usaspending-db-migrate:
+  usaspending-manage:
     build: .
     volumes:
      - .:/dockermount
-    command: /bin/sh -c "python -u manage.py migrate"
+    command: /bin/sh -c "python -u manage.py shell"
     environment:
       DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
```

Comment:
Should add `ES_HOSTNAME` env var to this app as some management commands need it